tests/sentry/integrations/vsts/test_issues.py

```diff
@@ -212,6 +218,60 @@ def test_get_issue_url(self):
         url = self.integration.get_issue_url(work_id)
         assert url == 'https://fabrikam-fiber-inc.visualstudio.com/_workitems/edit/345'
 
+    @responses.activate
+    def test_should_resolve_active_to_resolved(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': 'Active',
+            'new_state': 'Resolved',
+        })
+        assert should_resolve is True
+
+    @responses.activate
+    def test_should_resolve_resolved_to_active(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': 'Resolved',
+            'new_state': 'Active',
+        })
+        assert should_resolve is False
+
+    @responses.activate
+    def test_should_resolve_new(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': None,
```

Comment:
What if `old_state` is not even set as a key?

Reply:
So, this is at the point where I've gotten the data and changed things around. I'm building that exact dict here: https://github.com/getsentry/sentry/pull/10359/files#diff-2593e1bf96fda2ecc3b309977a5feba2R166. Because I'm using `.get('oldValue')` I'll always get some value. And because I'm constructing a dict with those exact parameters, I can't conceive of a case where that would happen. But if it does... it should error.